content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+
+```{r}
+str(mod_mr)
+```
+
+This is a long set of things, and we'll not go through it all, but know first that any subset that has a <code>$</code> before it is accessible as such. For example: 
+
+```{r}
+mod_mr$coefficients
+```
+
+Will get us the coefficients. This is useful. We can also ask for the standard errors: 
+
+```{r}
+summary_ob <- summary(mod_mr)
+summary_ob$coefficients[,2]
+```
+
+Other things we often want are the $R^2$ value: 
+
+```{r}
+summary_ob$adj.r.squared
+```
+
+We might also want the AIC (discussed below):
+
+```{r}
+stats::AIC(mod_mr)
+```
+
+Or the negative log-likelihood (also discussed below): 
+
+```{r}
+stats::logLik(mod_mr)
+```
+
+And there are a variety of other model components we might be interested in, the goal is to identify where they are in the fitted model object. Often other, perhaps more obscure values, are extracted easily enough, and the specific solution is easily Google-able. 
+
+## Effect Estimation
+
+A few things now about model comparison and effect estimation, which are both large subject areas. A quick and dirty way of estimating if the effect of this additional parameter is of importance is getting a 95% Confidence interval around the estimate. So we see from our summary table that the estimate is 0.9749. Well, a 95% confidence interval is easy enough to construct, as it is stated simply as $\mu \pm 1.96 * SE$ where SE is the standard error, which we see we already have in our output. So to calculate this in R, we can actually extract the coefficient value and standard error easily like so:
+
+```{r}
+estimate <- mod_mr$coefficients["rec_year"]
+std_err <- summary_ob$coefficients[,2]["rec_year"]
+
+```
+
+If we wanted to print these values (i.e. say we were working in an R Markdown document and preparing a report for a class), we could print them into the text via:
+
+```{r}
+print(paste0("The coefficient estimate is: ", estimate, 
+      " and the standard error is: ", std_err))
+```
+
+But back to our 95% CI, we can take those two valuesand perform a couple of multiplications:
+
+```{r}
+upper <- estimate + 1.96 * std_err
+lower <- estimate - 1.96 * std_err
+```
+
+and then we would display our results as the estimate and then the 95% in brackets like so: 
+
+```{r}
+print(paste0("The estimate of the coefficient is ", estimate, " with a 95% 
+             confidence interval of (", lower, ",", upper, ")"))
+```
+
+Or if we don't need quite so many digits and wanted to round it to three decimal places:
+
+```{r}
+print(paste0("The estimate of the coefficient is ", round(estimate, 3), " with a 95% confidence interval of (", round(lower, 3), ", ", round(upper, 3), ")"))
+```
+
+Now what does this tell us? Well, we're actually seeing that the 95% CI does not overlap zero, and so therefore we could interpret this as a "significant" effect. Note we haven't done a p-value based hypothesis test here, but this is a different and valid way to determine if an effect is of note. 
```

Comment:
Yes, in this case. But it is not always true that CIs give you the same answer as p-values